### DISCUSSION TITLE
用rclone挂载的Aliyun盘

### DISCUSSION THREAD

**Discussion by oiNGho** [AlistGo/alist#852] [Q&A]
出现很多这样的 ERRO 是什么原因呀？这个正常吗？使用上倒是没什么问题的样子
![image](https://user-images.githubusercontent.com/101927848/160634906-494a27cb-42e4-4263-b5c5-ae80f2bf798a.png)

**Comment by xhofe** [comment]
正常。

**Reply by oiNGho** [reply]
好的，谢谢
